dxr/app.py

```diff
@@ -9,7 +9,7 @@
 from time import time
 from mimetypes import guess_type
 from urllib import quote_plus
-
+from datetime import datetime
```

Comment:
As per PEP 8, we keep our imports in 3 stanzas: https://www.python.org/dev/peps/pep-0008/#imports. And also, alphabetize them within the stanzas to make it easier to look for things and harder to duplicate things.